### DISCUSSION TITLE
Unable to create table via APIs [Jun 06]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2349] [Discord Threads]
> 🙋‍♂️ **djang0**

**Comment by o1lab** [comment]
> 🙋‍♂️ **FrancyMak**  
#### Hi! Could you provide us the project info?  
#  
> 🙋‍♂️ **FrancyMak**  
#### And also how can we reproduce this error and how do you arrive to this message. Thank you  
#  
> 🙋‍♂️ **djang0**  
#### Hi <@234236409429491712> , I just created a project and want to create a table using https://all-apis.nocodb.com/#tag/DB-table/operation/db-table-create but it didn't works  
#  
> 🙋‍♂️ **djang0**  
#### I pasted the test json to try  
#  
> 🙋‍♂️ **FrancyMak**  
#### Alright.   
####    
#### Could you send the endpoint and the sending json?  
#  
> 🙋‍♂️ **FrancyMak**  
#### also the request, redact personal info  
#  
> 🙋‍♂️ **djang0**  
####   
![image0](https://cdn.discordapp.com/attachments/983068170770345984/983093260945854504/unknown.png)  
#  
> 🙋‍♂️ **djang0**  
#### i'm using xc-auth  
#  
> 🙋‍♂️ **FrancyMak**  
#### are you using a user with editor capabilities?  
#  
> 🙋‍♂️ **djang0**  
#### yes  
#  
> 🙋‍♂️ **djang0**  
#### this is the one i use on the gui  
#  
> 🙋‍♂️ **djang0**  
#### in fact I even have other problems, when I call the api to insert a line with a valid json, it creates a blank line, so I do not know if it is my version that is defective (latest) or not 🤔  
#  
> 🙋‍♂️ **djang0**  
#### it seems really instable  
#  
> 💁 **o1lab**  
#### editor is only allowed to change the data and not the schema  
#  
> 💁 **o1lab**  
#### you would need higher permissions  
#  
> 🙋‍♂️ **djang0**  
#### i'm using the token from creator and superadmin user  
#  
> 💁 **o1lab**  
#### Can u raise an issue with all details pls  
#  
> 💁 **o1lab**  
#### Something not right  
#  
> 💁 **o1lab**  
#### With the API being invoked on ur end  
#  
> 💁 **o1lab**  
#### If the token n API is same in GUI. There shouldn't be much issue  
#  
> 🙋‍♂️ **djang0**  
#### Yes it seems really buggy  
#  
> 🙋‍♂️ **djang0**  
#### ok thanks  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/983068170770345984)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
